build/Version.props

```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <VersionMajor>2</VersionMajor>
     <VersionMinor>2</VersionMinor>
-    <VersionPatch>205</VersionPatch>
-    <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">preview</ReleaseSuffix>
+    <VersionPatch>207</VersionPatch>
+    <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">rtm</ReleaseSuffix>
```

Comment:
@dsplaisted @nguerrera can you take a look while Livar is out?

Reply:
Looking.

Reply:
I think "servicing" matches the convention we've been asking about.

I never saw us use "rtm". You chose it here: https://github.com/dotnet/cli/pull/10175/files do you remember the reason?

I'm not sure it matters much since we don't ship actual previews of these servicing releases.

Reply:
> I think "servicing" matches the convention we've been asking about.
> 
> I never saw us use "rtm". You chose it here: https://github.com/dotnet/cli/pull/10175/files do you remember the reason?
> 
> I'm not sure it matters much since we don't ship actual previews of these servicing releases.

No idea.  Mainly asking about the preview vs. rtm case.  I didn't think we shipped any previews of these releases.